src/models/MyAccount.ts

```diff
@@ -249,9 +249,13 @@ export default class MyAccount {
         notificationPreferenceMap[patron.fixedFields["268"].value]
       return {
         notificationPreference,
-        name: patron.names[0],
+        name: MyAccount.formatPatronName(patron.names[0]),
```

Comment:
When we rolled out the holdshelf emails we opted to leave patron names as LAST, FIRST because it's really hard to parse names automatically (I'm sure you've seen [this blog post](https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/)). I imagine we would want to do the same here?

Reply:
Yes, I was definitely worried about this. Can you check the thread regarding this in the QA doc? https://docs.google.com/document/d/1rx8pHljNYFhItXtpEjaKPB9MHtqTfdxeEs7Nw8QpEuM/edit

I got the impression that it was possible based on confirmation emails that are sent. Can we add a middle name and second last name to a test patron to check this update?